modules/combined/tests/generalized_plane_strain_tm_contact/generalized_plane_strain_tm_contact_field.i

```diff
@@ -94,9 +94,9 @@
 
 [ScalarKernels]
   [./gps]
-    type = GeneralizedPlaneStrainDiag
+    type = GeneralizedPlaneStrain
     variable = strain_zz
-    gps_uo = gpsuo
+    generalized_plane_strain_userobject = gpsuo
```

Comment:
We are moving away from naming parameters `..._userobject`. It is already encoded in the parameter type that a _userobject_ name is expected.